photon-lib/src/main/native/cpp/photon/PhotonCamera.cpp

```diff
@@ -117,13 +128,9 @@ PhotonCamera::PhotonCamera(nt::NetworkTableInstance instance,
   HAL_Report(HALUsageReporting::kResourceType_PhotonCamera, InstanceCount);
   InstanceCount++;
 
-  {
-    std::lock_guard lock{g_timeSyncServerMutex};
-    if (!g_timeSyncServerStarted) {
-      timesyncServer.Start();
-      g_timeSyncServerStarted = true;
-    }
-  }
+  // Not sure if this is safe or not, since PhotonCamera can still be
+  // constructed statically by user code?
```

Comment:
TODO: this actually doesn't change logic. Users can still put `PhotonCamera camera{"lifecam"};` in their robot code's header and this'll be constructed statically right? So still a problem?